core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -799,6 +807,10 @@ private void loadMetadata(LockedInodePath inodePath, LoadMetadataContext context
           e);
       throw new IOException(e);
     }
+    if (failedSync > 0) {
+      throw new IOException("Failed to load metadata of " + failedSync
+              + " files or directories.");
```

Comment:
```suggestion
      throw new IOException(String.format("Failed to load metadata of %s files or directories under %s", failedSync, path));
```

Reply:
For this PR, I think it's fine to print the count, although this count may get confusing in the log message. We will address the confusion in future PRs.